### ISSUE TITLE
Update typedoc Peer Dependency

### ISSUE THREAD

**Issue by nyan-left** [rafern/typedoc-plugin-external-resolver#2]
Hello,

The peer dependency for typedoc in the project appears outdated ([package.json](https://github.com/rafern/typedoc-plugin-external-resolver/blob/main/package.json#L38)). Updating to the latest typedoc version would enhance compatibility and feature integration.

Could we bump the typedoc version to the latest? This would greatly benefit users of this plugin.

Thanks for your work on this project!

**Comment by rafern** [comment]
Hi, thanks for the heads-up. I've released a new version that is compatible with TypeDoc 0.25.x, as well as TypeScript 0.5.x. You probably already know this (I didn't when updating the package, and it caused some frustration), but make sure to add the `--plugin typedoc-plugin-external-resolver` option to typedoc, otherwise the plugin will just be ignored. I've updated the README to reflect these changes
